src/backend/core/api/viewsets.py

```diff
@@ -376,3 +377,38 @@ def perform_update(self, serializer):
                 raise exceptions.ValidationError({"role": message})
 
         serializer.save()
+
+
+class InvitationViewset(
+    mixins.CreateModelMixin,
+    viewsets.GenericViewSet,
+):
+    """API ViewSet for user invitations to team.
+
+    POST /api/v1.0/teams/<team_id>/invitations/ with expected data:
+        - email: str
+        - role: str [owner|admin|member]
+        - issuer : User, automatically added from user making query, if allowed
+        - team : Team, automatically added from requested URI
```

Comment:
Maybe shouldn't be listed here since this is "expected data" so the expected request payload?

Reply:
```suggestion
        (issuer and team are automatically added, from request user and URI)
```
You're right. Is this better ? I feel like this modification should be explicited in docs